salt/orchestrate/edx/cloudfront.sls

```diff
@@ -0,0 +1,113 @@
+{% set ENVIRONMENT = salt.environ.get('ENVIRONMENT') %}
+{% set env_dict = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set env_settings = env_dict.environments[ENVIRONMENT] %}
+{% set VPC_NAME = salt.environ.get('VPC_NAME', env_settings.vpc_name) %}
+{% set BUSINESS_UNIT = salt.environ.get('BUSINESS_UNIT', env_settings.business_unit) %}
+
+{% for purpose, settings in env_settings.purposes.items() %}
+{% if 'live' in purpose %}
+{% set purpose_prefix = purpose.strip('-live').strip('-draft') %}
+{% set name = [purpose_prefix, ENVIRONMENT, 'cdn']|join('-') %}
+{% set purpose = distribution.pop('purpose', env_settings.purpose) %}
+{% set business_unit = distribution.pop('business_unit', BUSINESS_UNIT) %}
+provision_cloudfront_distribution_for_{{ purpose }}_in_{{ ENVIRONMENT }}:
+  boto_cloudfront.present:
+    - name: {{ name }}
+    - tags:
+        Environment: {{ ENVIRONMENT }}
+        Name: {{ name }}
+        business_unit: {{ business_unit }}
+        Department: {{ business_unit }}
+        OU: {{ business_unit }}
+        purpose: {{ purpose }}
+    - config:
+        Enabled: true
+        Aliases:
+          Quantity: 0
+        CacheBehaviors:
+          Quantity: 0
+        CallerReference: '1528233024979'
```

Comment:
Should this be some random number maybe generated by vault?

Reply:
We should probably just delete that value...